model/database/registry.yaml

```
@@ -90,6 +101,29 @@ groups:
 
           This attribute has stability level RELEASE CANDIDATE.
         examples: ["someval", "55"]
+      - id: db.query.summary
+        type: string
+        stability: experimental  # RELEASE CANDIDATE
+        brief: >
+          Low cardinality representation of a database query text.
+        note: >
+          `db.query.summary` provides static summary of the query text. It describes
+          a class of database queries and is useful as a grouping key,
+          especially when analyzing telemetry for database calls involving complex queries.
+
+          Summary may be available to the instrumentation through SQL comment,
+          instrumentation hooks, or other means. If it is not available, instrumentations
```

(similar thought here)
```suggestion
          Summary may be available to the instrumentation through
          instrumentation hooks or other means. If it is not available, instrumentations
```